### PR TITLE
feat: user leaderboard page

### PR DESCRIPTION
**Pull request by omBratteng**
AS-358

## TODO

Added [users page](https://as-358-users-directory.preview.app.daily.dev/users) 

- [x] Check the width of score columns with real data

Laptop:
image

Tablet:
image

Mobile:
image

### Preview domain
https://as-358-users-directory.preview.app.daily.dev

### CODE REVIEW

File: packages/webapp/pages/users.tsx [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+            href={item.user.permalink}
+          >
+            <>
+              <span className="pl-1">{indexToEmoji(i)}</span>
+              <UserHighlight
+                {...item.user}
+                showReputation
+                className={{
+                  wrapper: 'min-w-0 flex-shrink !p-2',
+                  image: '!h-8 !w-8',
```
Comment (rebelchris) [inline packages/webapp/pages/users.tsx:79]:
```suggestion
                  image: '!size-8',
```

**Review by rebelchris — APPROVED** [review]
Amazing :D
Happy to see we could reuse so many things 🤯

File: packages/shared/src/hooks/useActiveNav.tsx [hunk 2]
```diff
@@ -33,6 +33,7 @@ export default function useActiveNav(activeFeed: AllFeedPages): UseActiveNav {
       OtherFeedPage.Tags,
       OtherFeedPage.Sources,
       OtherFeedPage.Leaderboard,
+      'users',
```
Comment (nensidosari) [inline packages/shared/src/hooks/useActiveNav.tsx:36]: Isn't the leaderboard mentioned above this one?

**Review by nensidosari — APPROVED** [review]
Looks great!! just one small thing there

**Pull request merged**